<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interpolation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="baseSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,7">
    <item row="0" column="2" rowspan="7">
     <widget class="QStcePlot" name="plotWidget" native="true"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="exit_button">
      <property name="text">
       <string>Beenden</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="reset_button">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="Interpolatiosart">
      <property name="title">
       <string>Interpolationsart</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QComboBox" name="interpolation_select">
         <item>
          <property name="text">
           <string>Linear</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Polynom</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Kubischer Spline</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="Definitionsbereich">
      <property name="title">
       <string>Definitionsbereich:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,0,1">
       <item row="0" column="3">
        <widget class="QLineEdit" name="xmax_edit">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="xmax_label">
         <property name="text">
          <string>xmax:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="ymax_label">
         <property name="text">
          <string>ymax:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="ymin_label">
         <property name="text">
          <string>ymin:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="ymax_edit">
         <property name="text">
          <string>50</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="ymin_edit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="xmin_edit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="xmin_label">
         <property name="text">
          <string>xmin:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QPushButton" name="defbereich_button">
         <property name="text">
          <string>Defnitionsbereich setzen</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QStcePlot</class>
   <extends>QWidget</extends>
   <header>qStcePlot/qstceplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
